get_webpage.py

```diff
@@ -155,6 +193,15 @@ def get_instpvs(url):
                     "TOTALCOUNTS", "DAETIMINGSOURCE", "MONITORCOUNTS", "MONITORSPECTRUM", "MONITORFROM", "MONITORTO",
                     "NUMTIMECHANNELS", "NUMSPECTRA"]
 
+    #rc_pvs = []
+    print blocks_all.values()[0]
```

Comment:
Print statements should be removed or modified into logger calls. I'm guessing these can just be removed.